sql/testdata/datetime

```diff
@@ -42,6 +42,99 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
```

Comment:
we need another test case where the time zone isn't set and the user provides a timestamp without a time zone, which results in an error, i think.